scripts/types_generator/formatter.py

```diff
@@ -1,17 +1,35 @@
+import re
+
 from datamodel_code_generator.format import CustomCodeFormatter
 
+from types_generator.config import ExtractorConfig
+
 
 class CodeFormatter(CustomCodeFormatter):
     """
     Make modifications to generated model.
     1. Remove unused 'ConfigDict' import
     2. Remove source filename info
+    3. Fix private class names starting with underscore _
+    4. Make all Query and Request classes generated from operation ID private
     """
 
     def apply(self, code: str) -> str:
-        return (
+        code = (
             code.replace("from pydantic import ConfigDict\n", "")
             .replace(" ConfigDict,", "")
             .replace("ConfigDict,", "")
             .replace(", ConfigDict", "")
         )
+
+        # datamodel-codegen cannot generate classname with underscore, because class name resolver contains this code:
```

Comment:
This is hacky, I could not find a better way to convince the generator to generate classnames with `_`